fuzzing/fuzz-targets/fuzz_config.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
```

Comment:
I'll remove this and the executable but on these fuzz harnesses files. This was mostly a copy paste oversight when I moved them. I intentionally documented the invocation of these tests using `python` instead of relying on the shebang in the README example to minimize the possibility of confusion for users in edge cases such as alternatively named python interpreter commands.

> Except under very strange and rare circumstances

You've peaked my curiosity, but that's a conversation for a different thread I suppose lol.